docs/metrics/pearson-correlation.md

```diff
@@ -0,0 +1,54 @@
+# Pearson Correlation Coefficient
+
+Pearson's correlation coefficient is a statistical measure that evaluates the strength and direction of the linear
+relationship between two continuous variables. The values range from -1 to +1. A value close to +1 indicates a strong
+positive linear correlation, where as one variable increases, the other also increases proportionally. A value close
+to -1 signifies a strong negative linear correlation, meaning as one variable increases, the other decreases
+proportionally. A value of 0 indicates no linear correlation between the variables.
+
+## Implementation Details
+
+The correlation coefficient is calculated by dividing the covariance of $x$ and $y$ by their individual standard
+deviations. This can be mathematically represented as:
+
+$$
+r = \frac{\text{cov}(x, y)}{\sigma_x \sigma_y} = \frac{\sum_{i=1}^{N} (x_i - \bar{x})(y_i - \bar{y})}{\sqrt{\sum_{i=1}^{N}
+(x_i - \bar{x})^2 \sum_{i=1}^{N} (y_i - \bar{y})^2}}
+$$
+
+where $x$ and $y$ are the collection of data points.
+
+### Examples
+
+Temperature Correlation:
+
+<div class="grid" markdown>
+
+| Ground Truth Temperature (&deg;C) | Predicted Temperature (&deg;C) |
+| --- | --- |
+| 25  | 30  |
+| 35  | 28  |
+| 30  | 27  |
+| 40  | 35  |
```

Comment:
I'd recommend removing one of these pairs to make the sample calculations easier to read.

Reply:
Same comment for the other doc too.